rules-java-diva/addon/src/main/java/org/jboss/windup/rules/apps/diva/analysis/DivaLauncher.java

```diff
@@ -0,0 +1,613 @@
+package org.jboss.windup.rules.apps.diva.analysis;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.Stack;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.jar.JarFile;
+import java.util.logging.Logger;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.__;
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.config.operation.GraphOperation;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.ArchiveModel;
+import org.jboss.windup.graph.model.ProjectDependencyModel;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.WindupConfigurationModel;
+import org.jboss.windup.graph.model.WindupFrame;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.graph.model.resource.SourceFileModel;
+import org.jboss.windup.graph.service.FileService;
+import org.jboss.windup.graph.service.GraphService;
+import org.jboss.windup.graph.service.WindupConfigurationService;
+import org.jboss.windup.rules.apps.diva.EnableTransactionAnalysisOption;
+import org.jboss.windup.rules.apps.diva.model.DivaAppModel;
+import org.jboss.windup.rules.apps.diva.model.DivaConstraintModel;
+import org.jboss.windup.rules.apps.diva.model.DivaContextModel;
+import org.jboss.windup.rules.apps.diva.model.DivaEndpointModel;
+import org.jboss.windup.rules.apps.diva.model.DivaEntryMethodModel;
+import org.jboss.windup.rules.apps.diva.model.DivaRequestConstraintModel;
+import org.jboss.windup.rules.apps.diva.model.DivaRequestParamModel;
+import org.jboss.windup.rules.apps.diva.model.DivaRestApiModel;
+import org.jboss.windup.rules.apps.diva.model.DivaRestCallOpModel;
+import org.jboss.windup.rules.apps.diva.service.DivaEntryMethodService;
+import org.jboss.windup.rules.apps.java.archives.model.IgnoredArchiveModel;
```

Comment:
```suggestion
import org.jboss.windup.graph.model.IgnoredArchiveModel;
```
Another PR that's just been merged has changed the location of this class